backend/runners/load_representative_points.py

```diff
@@ -164,10 +196,21 @@ def _get_all_service_areas(features):
             'state': state,
             'zip_code': zip_code,
             'location': postgis.to_polygon(geometry),
-            'state_fips': service_area_to_fips_codes[(state, county, zip_code)]['state_fips'],
-            'county_fips': service_area_to_fips_codes[(state, county, zip_code)]['county_fips'],
-            'nchs_urban_rural_code': urban_rural_designation,
-        })
+            'state_fips': service_area_to_fips_codes[(state, county, zip_code)].get(
```

Comment:
`TypeError: NoneType has no attribute '.get'`

Reply:
This should be ok now, no?